frontend/src/components/user-management/admin/AddAdminModal.tsx

```diff
@@ -127,7 +127,7 @@ const AddAdminModal = (): React.ReactElement => {
       </Button>
       <Modal isCentered isOpen={isOpen} onClose={onModalClose} size="3xl">
         <ModalOverlay />
-        <ModalContent maxW="80vw" p={2}>
+        <ModalContent borderRadius="12px" maxW="80vw" p={2}>
```

Comment:
`AddStudentModal` and `AddClassroomModal` also need this border radius

can we also refactor `MultiOptionModal` to use the generic modal and delete `ResponseTypeModal`?

Reply:
omg how did this `ResponseTypeModal` come back from the grave 👻